specs/withdrawals.md

```diff
@@ -18,12 +18,17 @@ an L2 account to an L1 account.
 more specific terms to differentiate:
 
 - A _withdrawal initiating transaction_ refers specifically to a transaction on L2 sent to the Withdrawals predeploy.
+- A _withdrawal proofing transaction_ refers specifically to an L1 transaction
```

Comment:
```suggestion
- A _withdrawal proving transaction_ refers specifically to an L1 transaction
```
given we're using `proving` below.

Reply:
Also the word `proofing` means something different than `proving`

Reply:
I agree, thank you.